cockroachdb/manifests-cert-manager/client.yaml

```
@@ -4,7 +4,7 @@ metadata:
   name: &app cockroachdb-client
   annotations:
     "app.uw.systems/description": "Used to connect and query the cockroachdb databases."
-    "app.uw.systems/repos.cockroachdb-manifests": "https://github.com/utilitywarehouse/cockroachdb-manifests"
+    "app.uw.systems/repos": "https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/cockroachdb"
```

Should we add an ID here? I don't think it's used by OpsLevel but might be useful to avoid this value being stomped if e.g. for some reason someone adds another `app.uw.systems/repos` annotation downstream? Do we have a best practice for this?

```suggestion
    "app.uw.systems/repos.shared-kustomize-bases": "https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/cockroachdb"
```

---

I like the philosophy of not adding things unless we need them now. 
In the hypothetical situation where somebody will need to add additional repository there, there won't be problem too- they can always use ID. 

> Do we have a best practice for this?

We don't, all that we have is [this doc](https://wiki.uw.systems/posts/ops-level-nz4v4ka0#hqoo3-app-uw-systems-repos-id)